src/patterns/dosdp-patterns/avoidance_of_food_intake.yaml

```diff
@@ -0,0 +1,67 @@
+pattern_name: avoidance_of_food
+pattern_iri: http://purl.obolibrary.org/obo/maxo/patterns/avoidance_of_medical_action.yaml
+description: ""
```

Comment:
There should be a reasonable explanation here along the line of what was discussed in the ticket.